core/eolearn/core/eoexecution.py

```diff
@@ -176,15 +193,15 @@ def _run_execution(self, processing_args: List[_ExecutorProcessingArgsType], wor
             MULTIPROCESSING_LOCK = None
 
     @classmethod
-    def _try_add_logging(cls, log_path: Optional[str], filter_logs_by_thread: bool,
-                         logs_filter: Optional[Filter]) -> Tuple[Optional[Logger], Optional[Handler]]:
+    def _try_add_logging(cls, log_path: Optional[str], filter_logs_by_thread: bool, logs_filter: Optional[Filter],
+                         logs_handler_factory: _LogsHandlerFactoryType) -> Tuple[Optional[Logger], Optional[Handler]]:
         """ Adds a handler to a logger and returns them both. In case this fails it shows a warning.
         """
         if log_path:
             try:
                 logger = logging.getLogger()
                 logger.setLevel(logging.DEBUG)
-                handler = cls._get_log_handler(log_path, filter_logs_by_thread, logs_filter)
+                handler = cls._get_log_handler(log_path, filter_logs_by_thread, logs_filter, logs_handler_factory)
```

Comment:
i would change this to `_build_log_handler` or something like that. Getting a handler would be the call to the factory, no?

Reply:
Done